server/src/Packages/Stripe/Resources/js/stripe.js

```diff
@@ -0,0 +1,108 @@
+import $ from 'jquery';
+import successTemplate from './templates/successTemplate';
+import errorTemplate from './templates/errorTemplate';
+
+class StripeService {
+    constructor() {
+        this.card = '';
+
+        try {
+            this.stripe = Stripe('pk_test_U2JalAfKOyR5DHS7R4KFeJLh00AdOsjkgo');
```

Comment:
The Stripe api key can be injected from the original config object, similar to how I made some modifications to insert the graphQl api URI into the scripts.

Reply:
@luke-siedle I implemented this